client/my-sites/themes/theme-showcase.jsx

```diff
@@ -284,8 +285,69 @@ class ThemeShowcase extends React.Component {
 						trackScrollPage={ this.props.trackScrollPage }
 						emptyContent={ this.props.emptyContent }
 					/>
-					<ThemePreview />
-					{ this.props.children }
+
+					<div className="theme-showcase__open-showcase-button-holder">
+						{ isShowcaseOpen ? (
```

Comment:
Should this be separated into a new component, and then we show `RecommendedThemes` along with `AllThemes`?

that could also be used if we need to gate the feature to certain users

Reply:
Perhaps it should.  These were all in the ThemeShowcase from the beginning, and not being 100% certain which components may require being hidden or extracted as part of `AllThemes` I haven't taken any steps in that direction yet.  

It's currently also the case that `RecommendedThemes` requires the `AllThemes` part of the code to run to maintain all of its functionality.  This is why I am hiding the `AllThemes` section with `display: none` instead of conditionally loading it in the first place.  When I had it conditioned to not load until the button is pushed, `RecommendedThemes` would have small bugs in its functionality until `AllThemes` was opened and loaded.

I am going to investigate a bit more to try and clean this all up, and in that case creating the `AllThemes` component could be good as well.  At its current state it works for how we want it to work (alongside `AllThemes`), but is lacking in the sense that `Recommended Themes` is not completely stand-alone and would cause issues down the line if we tried to use it somewhere without the `AllThemes` portion.